libs/test-setup/src/lib.rs

```diff
@@ -32,7 +32,7 @@ pub struct TestApiArgs {
     pub connector_tags: BitFlags<Tags>,
```

Comment:
These functions are getting a bit frankensteiny, but we might be able to remove them in the not too distant future, so :+1: